lib/msal-browser/test/error/NativeAuthError.spec.ts

```diff
@@ -58,6 +58,17 @@ describe("NativeAuthError Unit Tests", () => {
                 expect(error.errorCode).toBe("interaction_required");
             });
 
+            it("translates ACCOUNT_UNAVAILABLE status into corresponding InteractionRequiredError", () => {
+                const error = NativeAuthError.createError("interaction_required", "interaction is required", {
+                    error: 1,
```

Comment:
What is the "1" here?  Just a boolean true?  Or is this an error code.

Reply:
error code